libs/designsystem/src/lib/scss/print/generic/_typography.scss

```diff
@@ -1,11 +1,18 @@
 @media print {
   :root {
     --kirby-font-family: 'Roboto', sans-serif;
+    --font-size-xxlarge: 27pt;
```

Comment:
I'm not quite sure - but I think most other places we use custom properties they're prepended with `--kirby` to signify where they're coming from. Other than that it is looking good much more readable 😄!